cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
The individual molecules shouldn't know if they're around other molecules, so shouldn't know about their margin. That should be set one level above them where it's controlled where they appear. For example,

https://github.com/cfpb/cfgov-refresh/blob/spacingfix/cfgov/unprocessed/css/molecules/image-text-50-50.less#L87

Should actually be defined in `.o-image-text-50-50-group`